exp/api/v1beta1/gcpmanagedcluster_webhook.go

```diff
@@ -53,10 +57,37 @@ func (r *GCPManagedCluster) ValidateCreate() error {
 }
 
 // ValidateUpdate implements webhook.Validator so a webhook will be registered for the type.
-func (r *GCPManagedCluster) ValidateUpdate(old runtime.Object) error {
+func (r *GCPManagedCluster) ValidateUpdate(oldRaw runtime.Object) error {
 	gcpmanagedclusterlog.Info("validate update", "name", r.Name)
-
-	return nil
+	var allErrs field.ErrorList
+	old := oldRaw.(*GCPManagedCluster)
+
+	if !reflect.DeepEqual(r.Spec.Project, old.Spec.Project) {
```

Comment:
In CAPA we moved to using [go-cmp](https://github.com/google/go-cmp). How would you feel about changing these webhooks to use that?

Reply:
That sounds good to me. Updated the code to use go-cmp.

Reply:
Thanks